INSTALL.md

```diff
@@ -26,33 +25,50 @@ source code.  Unpack as follows.
     tar xzf <some dir>/accumulo-X.Y.Z-bin.tar.gz
     cd accumulo-X.Y.Z
 
+There are three scripts in the the `bin/` directory that are used to manage Accumulo:
+
+1. `accumulo` - Runs Accumulo command-line tools and starts Accumulo processes
+2. `accumulo-service` - Runs Accumulo processes as services
+3. `accumulo-cluster` - Manages Accumulo cluster on a single node or several nodes
+
+These scripts will be used in the remaining instructions to configure and run Accumulo.
+
+Add the following to your `~/.bashrc` if you want to be able to execute these scripts
+from any directory:
+
+    export PATH=/path/to/accumulo-X.Y.Z/bin:$PATH
+
+Source your `~/.bashrc` for the changes to take effect and verify:
+
+    source ~/.bashrc
+    which accumulo
+
+## Configuring
+
 Accumulo has some optional native code that improves its performance and
-stability.  Before configuring Accumulo attempt to build this native code
+stability.  Before configuring Accumulo, attempt to build this native code
 with the following command.
 
-    ./bin/build_native_library.sh
-
-If the command fails, its ok to continue with setup and resolve the issue
-later.
+    accumulo native
```

Comment:
After looking at the documentation, I think `create-config` is more clear.  Does that work?